templates/footer.html

```diff
@@ -0,0 +1,4 @@
+<div class="docs-rs-footer">
+    <a href="https://foundation.rust-lang.org/policies/privacy-policy/#docs.rs">Privacy policy</a>
+    <a href="https://foundation.rust-lang.org/">Rust Foundation</a>
```

Comment:
Hmm, is there a reason to link to the foundation's home page here?

Reply:
Just policy seemed a bit weird. So I just said to myself: "why not?".

Reply:
@GuillaumeGomez I would rather you moved the 'about' links to the footer instead: https://github.com/rust-lang/docs.rs/pull/1367#issuecomment-822051232

Reply:
Fine by me! Gonna need a little reword on the top navbar resize then.

Reply:
After taking a look, I don't think we should move "about" from the topbar: it provides information that are required when you want to customize your "docs.rs experience". It's not "extra info" like what we put in the footer.

Reply:
If the issue is the "Rust foundation" link, I can simply remove it. :)

Reply:
Honestly I would remove both the "About" and "Releases" links from the navbar, and have the footer be:

```
About docs.rs | Privacy Policy | Build queue
```

I looked at the stats we have on Grafana for the past 7 days:

* The whole about section got 0.0045% of the visits
* The whole release section except for the build queue got 0.0023% of the visits
* The build queue page got 0.0008% of the visits

I don't think either of the links in the navbar are that used to warrant their positioning.

Reply:
Ok, let's go with that then.